service/src/test/java/bio/terra/workspace/app/controller/WorkspaceApiControllerTest.java

```diff
@@ -147,7 +147,7 @@ public void createWorkspace_duplicateUuid_throws409() throws Exception {
         workspaceId,
         /*stageModel=*/ null,
         /*policies=*/ null,
-        HttpStatus.SC_CONFLICT);
+        HttpStatus.SC_BAD_REQUEST);
```

Comment:
I don't feel strongly about this but it is an interesting change. Does the response note why it's a bad request?

Reply:
I removed a piece of pre-check code that duplicates the check in the DAO. The workspace DAO throws `DuplicateWorkspaceException` that derives from `BadRequestException` with the error text:

`"Workspace with id %s already exists - display name %s stage %s"`